lib/main.dart

```diff
@@ -44,6 +44,8 @@ import 'package:provider/provider.dart';
 import 'package:rrule/rrule.dart';
 import 'package:time_machine/time_machine.dart';
 
+// FIXME: Our university website certificates have some issues, so we say we
```

Comment:
??

Reply:
it came from master?

Reply:
It did not, it's a duplicate of the comment below on HttpOverrides.

Reply:
oh, right, solved